ci/lib/install_functions.inc.sh

```diff
@@ -117,7 +117,7 @@ netbsd_install() {
 }
 
 linux_prepare_ribose_yum_repo() {
-  "${SUDO}" rpm --import https://github.com/riboseinc/yum/raw/master/ribose-packages.pub
```

Comment:
I'm tempted to add both public keys (`ribose-packages.pub` and `ribose-packages-next.pub`) so when time comes for another key update, these scripts do not need changes.

Reply:
@ribose-jeffreylau Thanks, fixed and re-pushed.